### PR TITLE
Add ubiquiti devices 

### PR DESCRIPTION
**Pull request by tpmeredith**
This pull request includes the following additions and updates to the Device-Type Library:
- Added YAML files for new Ubiquiti devices: 
    - USW-Pro-Max-24 
    - USW-Pro-Max-48-PoE
    - USW-Pro-Max-48
    - U7-Pro-Max
    - U7-Outdoor
    - USP-PDU-Pro
    - USP-PDU-HD
    - Enterprise Fortress Gateway (EFG)
    - Enterprise Campus Aggregation Switch (ECS Aggregation)
- Updated YAML for existing device (USW-Pro-Max-24-PoE) to include images and RPS power port.
- Uploaded elevation images for all new devices.

### CODE REVIEW

**Comment by harryajc** [comment]
Hi @tpmeredith 
For devices with hot swappable PSU these need to be modeled under `module-bays` rather than `power-ports` and then add the required PSU models in the Ubiquiti modules folder.

Many Thanks

Harry

**Comment by tpmeredith** [comment]
> Hi @tpmeredith For devices with hot swappable PSU these need to be modeled under `module-bays` rather than `power-ports` and then add the required PSU models in the Ubiquiti modules folder.
> 
> Many Thanks
> 
> Harry

Thank you for the feedback. I believe I accomplished this now.

File: device-types/Ubiquiti/USW-Pro-Max-24-PoE.yaml [hunk 1]
```diff
@@ -117,3 +119,6 @@ power-ports:
   - name: Input
     type: iec-60320-c14
     maximum_draw: 450
+  - name: USP-RPS Input
+    type: iec-60320-c20  # Standard connector for RPS
```
Comment (harryajc) [inline device-types/Ubiquiti/USW-Pro-Max-24-PoE.yaml:123]: This is not a iec-60320-c20 port please use 'ubiquiti-smartpower'

File: device-types/Ubiquiti/USW-Pro-Max-24.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - name: SFP+ 25
+    type: 10gbase-x-sfpp
+  - name: SFP+ 26
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: AC Input
+    type: iec-60320-c14
+    maximum_draw: 50
+  - name: USP-RPS Input
+    type: iec-60320-c20  # Standard connector for RPS
```
Comment (harryajc) [inline device-types/Ubiquiti/USW-Pro-Max-24.yaml:78]: This is not a iec-60320-c20 port please use 'ubiquiti-smartpower'

File: device-types/Ubiquiti/USW-Pro-Max-48-PoE.yaml [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  - name: SFP+ 51
+    type: 10gbase-x-sfpp
+  - name: SFP+ 52
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: AC Input
+    type: iec-60320-c14
+    maximum_draw: 820
+  - name: USP-RPS connector for PSU failover
+    type: iec-60320-c20  # Updated from usp-rps
```
Comment (harryajc) [inline device-types/Ubiquiti/USW-Pro-Max-48-PoE.yaml:227]: This is not a iec-60320-c20 port please use 'ubiquiti-smartpower'

File: device-types/Ubiquiti/USW-Pro-Max-48.yaml [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+  - name: SFP+ 51
+    type: 10gbase-x-sfpp
+  - name: SFP+ 52
+    type: 10gbase-x-sfpp
+power-ports:
+  - name: AC Input
+    type: iec-60320-c14
+    maximum_draw: 100
+  - name: USP-RPS Input
+    type: iec-60320-c20  # Updated to reflect standard support
```
Comment (harryajc) [inline device-types/Ubiquiti/USW-Pro-Max-48.yaml:130]: This is not a iec-60320-c20 port please use 'ubiquiti-smartpower'

**Review by harryajc — CHANGES_REQUESTED** [review]
Just a few changes are required to make the data as accurate as possible

File: device-types/Ubiquiti/USW-Pro-Max-48.yaml (hunk 4 above)
Comment (tpmeredith) [reply device-types/Ubiquiti/USW-Pro-Max-48.yaml]: I updated this to the ubiquiti-smartpower.

**Review by harryajc — APPROVED** [review]

**Pull request merged**